Annex60/Fluid/Movers/Examples/MoverParameter.mo

```diff
@@ -54,12 +54,13 @@ model MoverParameter
     redeclare package Medium = Medium,
     dynamicBalance=false,
     filteredSpeed=false,
-    speeds={0,1000,2000},
-    redeclare Annex60.Fluid.Movers.Data.Pumps.Wilo.Stratos25slash1to4 per,
-    inputType=Annex60.Fluid.Types.InputType.Constant,
-    speed=2000) "Pump with speed input"
+    redeclare Annex60.Fluid.Movers.Data.Pumps.Wilo.Stratos25slash1to4 per(
+      speeds_rpm={0,1000,2000},
+      constantSpeed_rpm=2000),
```

Comment:
I agree and added `speed_rpm_nominal` and `speed_nominal`. Now the unit tests are correct again.